### DISCUSSION TITLE
Mapping of Oleylamine (C18H35NH2)

### DISCUSSION THREAD

**Discussion by rashed-mhasan** [Martini-Force-Field-Initiative/MartiniDiscussions#10] [Polymers]
Hi,

I am new to Martini forcefield. I would like to map Oleylamine (C18H35NH2) for MD simulation.

Could anyone please help me understand how to assign martini beads to map Oleylamine (C18H35NH2)?

Thanks.

Best regards,
Rashed

**Comment by ricalessandri** [comment]
Please check out this tutorial: https://cgmartini.nl/docs/tutorials/Martini3/Small_Molecule_Parametrization/, it explains the steps necessary to parametrize a new molecule, like Oleylamine, within the Martini 3 framework.
